packages/jest-snapshot/src/snapshot_resolver.ts

```diff
@@ -64,9 +67,15 @@ function createDefaultSnapshotResolver(): SnapshotResolver {
 }
 
 function createCustomSnapshotResolver(
+  config: Config.ProjectConfig,
   snapshotResolverPath: Config.Path,
 ): SnapshotResolver {
-  const custom: SnapshotResolver = require(snapshotResolverPath);
+  const transformer = new ScriptTransformer(config);
+  const custom = interopRequireDefault(
+    transformer.requireAndTranspileModule<SnapshotResolver>(
```

Comment:
`interopRequireDefault` returns `any` (it probably shouldn't...), so you want your type annotation as `const custom: SnapshotResolver = ` as it was previously 🙂